README.md

```diff
@@ -84,6 +84,8 @@ These great products are built on Semantic UI React. Add yours [here][22].
 - https://thefaithcircle.com
 - https://appfollow.io
 - http://according.to.localsourc.es
+- http://www.aircip.ir
```

Comment:
This redirects to http://www.aircip.ir/auth/login, which is a 404.

Reply:
Sorry, because this project runs on IIS and IIS doesn't serve react-router URLs, it returns this error.